tests/unit/OptionsListUtilsTest.js

```diff
@@ -188,6 +188,28 @@ describe('OptionsListUtils', () => {
             unreadActionCount: 1,
         },
     };
+    
+    const REPORTS_WITH_MORE_PINS = {
+        ...REPORTS,
+        10: {
+            lastVisitedTimestamp: 1610666739302,
+            lastMessageTimestamp: 1,
+            isPinned: true,
+            reportID: 10,
+            participants: ['d_email@email.com'],
+            reportName: 'D report name',
+            unreadActionCount: 0,
+        },
+        11: {
+            lastVisitedTimestamp: 1610666732302,
+            lastMessageTimestamp: 1,
+            isPinned: true,
+            reportID: 11,
+            participants: ['z_email@email.com'],
+            reportName: 'Z Report Name',
+            unreadActionCount: 0,
+        },
```

Comment:
Let's change the id so that they follow incremental order.

Reply:
But after 9 is 10 right. I am continuing original `REPORTS` data.

Reply:
If you look at the previous report array they are following the same pattern and we can do the same

Reply:
okay